sample/src/main/kotlin/com/chuckerteam/chucker/sample/GraphQlTask.kt

```diff
@@ -6,23 +6,68 @@ import com.apollographql.apollo3.network.okHttpClient
 import kotlinx.coroutines.MainScope
 import kotlinx.coroutines.launch
 import okhttp3.OkHttpClient
+import retrofit2.Callback
+import retrofit2.Call
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.create
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Query
 
+private const val GRAPHQL_BASE_URL = "https://rickandmortyapi.com/graphql/"
+private const val BASE_URL = "https://rickandmortyapi.com/"
 class GraphQlTask (
     client: OkHttpClient,
 ) : HttpTask {
 
     private val apolloClient = ApolloClient.Builder()
-        .serverUrl("https://rickandmortyapi.com/graphql")
+        .serverUrl(GRAPHQL_BASE_URL)
         .okHttpClient(client)
         .build()
 
+    private val api = Retrofit.Builder()
+        .baseUrl(BASE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
```

Comment:
This converter factory is unnecessary as you're returning `Any?`

Reply:
Been getting the `java.lang.IllegalArgumentException: Unable to create converter for class java.lang.Object
        for method Api.getCharacterById` when I remove the converter factory.
        So, I'm now using `ResponseBody` instead.